src/settings/profile.hpp

```diff
@@ -62,9 +62,12 @@ class Profile
         return result;
     }
 
-    void add_per_file_settings(PerFileSettings settings) noexcept
+    void add_per_file_settings(PerFileSettings settings, bool front = false) noexcept
```

Comment:
Please use two separate functions (`append` and `prepend`?)

Reply:
Changed.